src/Confluent.Kafka/Producer.cs

```diff
@@ -341,12 +344,13 @@ public int Poll(int millisecondsTimeout)
         ///     Returns the number of events served.
         /// </returns>
         public int Poll(TimeSpan timeout)
-            => Poll(timeout.TotalMillisecondsAsInt());
-
+            => Poll(timeout.TotalMillisecondsAsInt());
+
         /// <summary>
-        ///     Poll for callback events. Only use this method on producer
-        ///     instances where background polling is disabled. Blocks
-        ///     until there is a callback event ready to be served.
+        ///     Poll for callback events. You will not typically need
```

Comment:
dito.

another thing: isnt it better to avoid all this doc duplication and simply reference the first one?